docker-compose.yml

```diff
@@ -6,10 +6,9 @@ services:
       - POSTGRES_USER=apilos
       - POSTGRES_PASSWORD=apilos
     volumes:
-      - ./pg-init-scripts:/docker-entrypoint-initdb.d
-      # Use: delegated to get better performances on docker write workloads where the sync with the host is not mandatory
-      - ./pgdata:/var/lib/postgresql/data:delegated
-      - ./pgconf/postgresql.conf:/etc/postgresql/postgresql.conf
+      - ./.docker/postgres/data:/var/lib/postgresql/data
```

Comment:
faut-il garder le `:delegated` ?

Reply:
Dans la mesure où ne le précise nulle part ailleurs je pense qu'on peut dire qu'on s'en fout, d'autant que [ça a l'air superflu désormais](https://forums.docker.com/t/what-happened-to-delegated-cached-ro-and-other-flags/105097)